src/hubdata/connect_hub.py

```diff
@@ -78,33 +79,100 @@ def __init__(self, hub_path: str | Path):
         self.model_output_dir = model_output_dir
 
 
-    def get_dataset(self) -> ds:
+    def get_dataset(self, ignore_files: Iterable[str] = ('README', '.DS_Store')) -> ds:
         """
+        Main entry point for getting a pyarrow dataset to work with. Prints a warning about any files that were skipped
+        during dataset file discovery.
+
+        :param: ignore_files a str list of file **names** (not paths) or file **prefixes** to ignore when discovering
+            model output files to include in dataset connections. Parent directory names should not be included. The
+            default is to ignore the common files `"README"` and `".DS_Store"`, but additional files can be excluded by
+            specifying them here.
         :return: a pyarrow.dataset.Dataset for my model_output_dir
         """
         # create the dataset. NB: we are using dataset "directory partitioning" to automatically get the `model_id`
-        # column from directory names
+        # column from directory names. regarding performance on S3-based datasets, we set `exclude_invalid_files=False`,
+        # which speeds up pyarrow's dataset processing, but opens the door to errors: "unsupported files may be present
```

Comment:
Is `exclude_invalid_files` set to `False` by default? Can the user override this?

Reply:
Yes, I decided that we'd never want it to be `True`, unlike the R version, which allows indirect control of that variable via `connect_hub()`'s `skip_checks` parameter. Maybe I don't understand the use case of wanting `exclude_invalid_files` to be `True`...

Reply:
OK, that helps. I will add `exclude_invalid_files` (defaulting to False) to `get_dataset()`'s params.